src/components/internal/SecuredFields/lib/core/utils/init/processAriaConfig.ts

```diff
@@ -0,0 +1,41 @@
+import { CSF_FIELDS_ARRAY, IFRAME_TITLE } from '../../../configuration/constants';
+import getProp from '../../../../../../../utils/getProp';
+import { addErrorTranslationToObject } from '../../../../utils';
+import { AriaConfigObject, AriaConfig } from '../../AbstractSecuredField';
+
+/**
+ * Checks if the merchant has defined an ariaConfig object and if so enhances it with a iframeTitle and error property, if they don't already exist
+ * If the ariaConfig object doesn't exist then creates one with the default properties
+ * In both cases where the error property doesn't exist we extract the correct translation for the field and use that
+ */
+export function processAriaConfig(configObj, fieldType, i18n) {
+    const iframeTitle: string = IFRAME_TITLE;
+    let newAriaFieldConfigObj: AriaConfigObject;
+    let ariaFieldConfigWithTranslation: AriaConfigObject;
+
+    // Check for a pre-existing, merchant defined object
+    const ariaFieldConfig: AriaConfigObject = getProp(configObj, `iframeUIConfig.ariaConfig.${fieldType}`);
+
+    if (ariaFieldConfig) {
+        newAriaFieldConfigObj = {
+            ...ariaFieldConfig,
+            // If object already has a title, use it - else set default
+            iframeTitle: ariaFieldConfig.iframeTitle || iframeTitle
+        };
+
+        // Add error translation
+        ariaFieldConfigWithTranslation = addErrorTranslationToObject(newAriaFieldConfigObj, fieldType, i18n, CSF_FIELDS_ARRAY);
+    } else {
+        // Create a new object with the default title
+        newAriaFieldConfigObj = { iframeTitle };
+        // Add error translation
+        ariaFieldConfigWithTranslation = addErrorTranslationToObject(newAriaFieldConfigObj, fieldType, i18n, CSF_FIELDS_ARRAY);
+    }
```

Comment:
Small suggestion: Since lines 27 and 32 are the same, you could delete line 14 and declare `ariaFieldConfigWithTranslation` as a `const` after the if/else.

Reply:
ok